### PR TITLE
Implement story 95 work party report search

### PR DESCRIPTION
**Pull request by alecpm**
This includes refactoring of the trail summary listing, and incorporation of Completed work parties in elastic search indexes (with once per day indexing).

### CODE REVIEW

**Comment by davisagli** [comment]
Are there corresponding changes for wta_buildout to link to the report search?

File: src/vms/report.py [hunk 1]
```diff
@@ -335,37 +354,50 @@ def related_reports(context, request):
...
+        for t in trails:
+            request.GET.extend({'trails': t})
 
     # Permission stuff
     groups = set(request.effective_principals or [])
     is_staff = 'g:Staff' in groups or 'g:Admin' in groups
     is_lm = 'g:Land Manager' in groups
+    lm_ids = [g[3:] for g in groups if g.startswith('lm:')]
+    for lm in lm_ids:
+        request.GET.extend({'land_mangers': lm})
```
Comment (davisagli) [inline src/vms/report.py]: s/mangers/managers/

File: src/vms/salesforce_client.py [hunk 2]
```diff
@@ -188,6 +188,43 @@ def list_published_workparties(self):
...
+            "Alert_ACLs_Needed__c, Alert_Low_Registration__c, Place_On_Waitlist__c, "
+            "Work_Party_Status__c, Group_WP_Open_To_Public__c, Incentive_Days__c, "
+            "Number_of_Crew_Leaders__c, Number_of_Assistant_Crew_Leaders__c, "
+            "Number_of_Registered_Volunteers__c, Number_of_Remaining_Spots__c, "
+            "Number_of_Youth_Registrations__c, Total_Participants_Registered__c, Sawyers_Needed__c, "
+            "Time_Exception__c, Time_Exception_Description__c, "
+            "Service_Hours_Total__c, Total_Volunteer_Work_Hours__c, "
+            "Breakfast_Spot__r.Name, Breakfast_Spot__r.city__c, Breakfast_Spot__r.directions__c "
+            "FROM Work_Party__c "
+            "WHERE Work_Party_Status__c='Complete'")
```
Comment (davisagli) [inline src/vms/salesforce_client.py]: We should really change this so we aren't duplicating the entire list of wp fields 3 times

**Comment by davisagli** [comment]
Needs a couple minor fixes, otherwise looks good

File: src/vms/report.py (hunk 1 above)
Comment (alecpm) [inline src/vms/report.py]: Good catch!  Thanks.

**Comment by alecpm** [comment]
Not yet.  Working on the wta_buildout updates.

**Comment by alecpm** [comment]
Fixes are in.  I've merged master in, including the salesforce query changes.

**Comment by davisagli** [comment]
deployed

**Pull request merged**